apps/pkcs12.c

```diff
@@ -820,16 +820,6 @@ int pkcs12_main(int argc, char **argv)
         }
     }
     if (macver) {
-        EVP_KDF *pkcs12kdf;
-
-        pkcs12kdf = EVP_KDF_fetch(app_get0_libctx(), "PKCS12KDF",
-                                  app_get0_propq());
-        if (pkcs12kdf == NULL) {
-            BIO_printf(bio_err, "Error verifying PKCS12 MAC; no PKCS12KDF support.\n");
-            BIO_printf(bio_err, "Use -nomacver if MAC verification is not required.\n");
-            goto end;
-        }
-        EVP_KDF_free(pkcs12kdf);
```

Comment:
It seems like we are going to lost good error message output in the vast majority of cases where PKCS12KDF is still in use. I assume what we get instead is something much more cryptic? Is there anyway to check up front what KDF is going to be used?

Reply:
Agreed. I've restored the check in case we don't use PBMAC1